<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SphericalMapTargetRegionWidget</class>
 <widget class="QWidget" name="SphericalMapTargetRegionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="mAzimuthGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Azimuth</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="UtQtAngleValueEdit" name="mAzimuthAngleMinEdit">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mAzimuthAngleLabel">
        <property name="text">
         <string>Azimuth Angle</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="mMinLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Minimum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="mMaxLabel">
        <property name="text">
         <string>Maximum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="UtQtAngleValueEdit" name="mAzimuthAngleMaxEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mElevationAngleLabel">
        <property name="text">
         <string>Elevation Angle</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="UtQtAngleValueEdit" name="mElevationAngleMinEdit"/>
      </item>
      <item row="3" column="3">
       <widget class="UtQtAngleValueEdit" name="mElevationAngleMaxEdit"/>
      </item>
      <item row="1" column="4">
       <widget class="UtQtAngleValueEdit" name="mAzimuthAngleStepEdit"/>
      </item>
      <item row="3" column="4">
       <widget class="UtQtAngleValueEdit" name="mElevationAngleStepEdit"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="mStepLabel">
        <property name="text">
         <string>Step</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="mAltitudeRangeGroupBox">
     <property name="title">
      <string>Altitude/Range</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QRadioButton" name="mAltitude_RB">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="mRange_RB">
        <property name="text">
         <string>Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="UtQtLengthValueEdit" name="mRangeLineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="UtQtLengthValueEdit" name="mAltitudeLineEdit"/>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtLengthValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtAngleValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mAzimuthAngleMinEdit</tabstop>
  <tabstop>mAzimuthAngleMaxEdit</tabstop>
  <tabstop>mAzimuthAngleStepEdit</tabstop>
  <tabstop>mElevationAngleMinEdit</tabstop>
  <tabstop>mElevationAngleMaxEdit</tabstop>
  <tabstop>mElevationAngleStepEdit</tabstop>
  <tabstop>mAltitude_RB</tabstop>
  <tabstop>mAltitudeLineEdit</tabstop>
  <tabstop>mRange_RB</tabstop>
  <tabstop>mRangeLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
